.buildkite/create_integration_testing_worksheet.py

```diff
@@ -6,61 +6,106 @@
 import os
 
 import gspread
 from oauth2client.service_account import ServiceAccountCredentials
 
 
 SPREADSHEET_CREDENTIALS = os.getenv("GOOGLE_APPLICATION_CREDENTIALS")
-SPREADSHEET_FILE_KEY = os.getenv("WKS_FILE_KEY")
+SPREADSHEET_TEMPLATE_KEY = "1kVhg0evo9EV2aDo10KdIIjwqsoT4rISR7dJzf6s_-RM"
+SPREADSHEET_TITLE = "Integration testing with Gherkin scenarios"
 
 SHEET_TAG = os.getenv("BUILDKITE_TAG")
 SHEET_TEMPLATE_FEATURES_COLUMN = 'B'
-SHEET_TEMPLATE_INDEX = 0
-SHEET_MAX_EMPTY_CELL = 20
-SHEET_TEMPLATE_TAG = 'release-v0.11.x'
+SHEET_INDEX = 0
+
+GIT_FEATURE_LINK = "https://github.com/learningequality/kolibri/blob/%s/integration_testing/features" \
+    % (SHEET_TAG)
 
 SCOPE = ['https://spreadsheets.google.com/feeds',
          'https://www.googleapis.com/auth/drive']
 
+GOOGLE_ACCOUNT = gspread.authorize(ServiceAccountCredentials.from_json_keyfile_name(SPREADSHEET_CREDENTIALS, SCOPE))
+
 
-def copy_worksheet():
-    google_account = gspread.authorize(ServiceAccountCredentials.from_json_keyfile_name(SPREADSHEET_CREDENTIALS, SCOPE))
-    wks_open = google_account.open_by_key(SPREADSHEET_FILE_KEY)
-    wks_id = wks_open.get_worksheet(SHEET_TEMPLATE_INDEX).id
-    return wks_open.duplicate_sheet(wks_id, insert_sheet_index=1, new_sheet_id=None, new_sheet_name=SHEET_TAG)
+def create_spreadsheet():
+    return GOOGLE_ACCOUNT.copy(SPREADSHEET_TEMPLATE_KEY, title=SPREADSHEET_TITLE, copy_permissions=True)
 
 
-def get_worksheet_link(wks_id):
+def get_worksheet_link(sheet_id, wks_id):
     wks_domain = "https://docs.google.com/spreadsheets/d/"
     link_arrg = "{0}{1}{2}{3}"
-    return link_arrg.format(wks_domain, SPREADSHEET_FILE_KEY, "/edit#gid=", wks_id)
-
-
-def edit_new_worksheet():
-    worksheet = copy_worksheet()
-    print(worksheet.id)
-    count_cell_empty = 0
-    for count in itertools.count():
-        current_cell = SHEET_TEMPLATE_FEATURES_COLUMN + str(count + 1)
-        cell_value = worksheet.acell(current_cell, value_render_option='FORMULA').value
-        edited_cell_value = cell_value.replace(SHEET_TEMPLATE_TAG, SHEET_TAG)
-        if SHEET_TAG in edited_cell_value:
-            count_cell_empty = 0
-            worksheet.update_acell(current_cell, edited_cell_value)
-        else:
-            if not str(edited_cell_value):
-                count_cell_empty += 1
-            if count_cell_empty >= SHEET_MAX_EMPTY_CELL:
-                break
-    wks_link = get_worksheet_link(worksheet.id)
-    print("Here's the new integration testing worksheet link =====>", wks_link)
+    return link_arrg.format(wks_domain, sheet_id, "/edit#gid=", wks_id)
+
+
+def get_feature_name(str_arg):
+    str_name = str_arg.replace("-", " ").replace(".feature", " ").capitalize()
+    return str_name.strip()
+
+
+def get_role_name(str_arg):
+    return str_arg.replace("-", " ").capitalize() + " scenarios"
+
+
+def get_feature_dir_path():
+    patent_path = os.path.dirname(os.getcwd())
+    return patent_path + "/integration_testing/features/"
```

Comment:
I don't understand what is `patent_path`?  Maybe a misspelling for `parent_path`? 😸